projects/cn0363/common/cn0363_pmod.txt

```diff
@@ -0,0 +1,12 @@
+Pin            Port               Schematic_name     System_top_name        IOSTANDARD  Termination
+
+# cn0363
+
+PMOD [2]       PMOD JA1           GAIN0              gain0_o                LVCMOS33    #N/A
```

Comment:
The ports and pins should be updated similarly to the pulsar project.

Reply:
Done.